torch/testing/_creation.py

```diff
@@ -14,12 +14,9 @@
     torch.complex64: torch.float32,
     torch.complex128: torch.float64,
 }
-float_to_corresponding_complex_type_map = {
-    v: k for k, v in complex_to_corresponding_float_type_map.items()
-}
 
 
-def _uniform_random(t: torch.Tensor, low: float, high: float):
```

Comment:
In accordance with other inplace ops, I postfixed this with an underscore.

Reply:
Sure